ideas/313-core-networking.md

```diff
@@ -0,0 +1,70 @@
+---
+id: #313-core-network-incentives
+title: Core Network Incentives Swarm Proposal
+status: research
+contributors:
+    - igor
```

Comment:
smart contract ppl? @3esmit @richard-ramos or @bgits

Reply:
added to "Additional Contributors" so far